<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NiippWidget</class>
 <widget class="QWidget" name="NiippWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QPushButton" name="_pb_enable_complex">
     <property name="text">
      <string>Включить комплекс</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_power">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Зона</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lbl_distance">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Радиус</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="_sb_power"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="_le_distance">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="_pb_start">
       <property name="text">
        <string>Пуск</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="_sl_power">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_antena">
       <property name="text">
        <string>Выбор антенны</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lbl_mode">
       <property name="text">
        <string>Режим работы</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QComboBox" name="_cb_antena">
       <item>
        <property name="text">
         <string>Направленная</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ненаправленная</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QComboBox" name="_cb_mode">
       <item>
        <property name="text">
         <string>Облучение</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Приведение</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Круговой обзор</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_l1">
       <property name="text">
        <string>Широта</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLabel" name="lbl_l2">
       <property name="text">
        <string>Долгота</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="_le_lat"/>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="_le_lon"/>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="_clear_uvod">
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lbl_status">
     <property name="text">
      <string>Статус</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="_le_status"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
